### ISSUE TITLE
[NPC] Leather working Supplies 'Missing Text' Error

### ISSUE THREAD

**Issue by Lupeli** [dalaranwow/dalaran-wow#4904]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: 

Jangdor Swiftstrider -  Leather working Supplies - in Camp Mojache. 

when spoken to, replies with 'Missing Text' and the option to buy. 

**Expected behaviour**:

To be greeted.

**Steps to reproduce the problem**:

1.  Fly to Camp Mojache in Feralas
2.  Speak to Jangdor Swiftswider in side the building
3. Note the Error

**Include proofs for this behaviour**

https://imgur.com/a/D89OV

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @Lupeli, thank you for the report.
I can confirm that on ptr. Regarding the issue I could find the texts only from Cata + videos. The text there is:
``` sql
Welcome to Stonemaul Hold, brave $C. How may I assist you this day?
```
I would assume the only changed part is the name of the place that he is at, so in the past it should of been:

``` sql
Welcome to Camp Mojache, brave $C. How may I assist you this day?
```

**Comment by Rushor** [comment]
Related: https://github.com/dalaranwow/dalaran-wow/issues/2731. I will close https://github.com/dalaranwow/dalaran-wow/issues/2731 for now. Thiss issue here has more details on the correct text.

**Comment by Rushor** [comment]
DB/Creature: Jangdor Swiftstrider - Add correct pagetext

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4904

The issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Just checked he text on PTR and it is incorrect, pre Cata should be the 2-nd one pointed above, because 'Stonemaul Hold' does not exist in Wotlk.
![image](https://user-images.githubusercontent.com/6797611/31652431-b62e5854-b327-11e7-9192-ebcbde0035f7.png)

**Comment by TeridanDW** [comment]
Same issue as above.

**Comment by Rushor** [comment]
thsi works fone on my local ptr. https://i.imgur.com/WDbW7Gv.jpg

so either fix wasn't apllied on ptr, or the cache of the client wasn't deleted before testing

**Comment by TeridanDW** [comment]
Works as intended on PTR. :)
